src/PresentationalComponents/Wizard/Wizard.js

```diff
@@ -72,7 +72,7 @@ class Wizard extends Component {
                     variant="primary"
                     isDisabled={ !isValidated }
                     onClick={ () => this.handleOnClose(true) }>
-                        Confirm
+                    { this.props.confirmAction }
```

Comment:
Please do not reference confirm action trough `this.props` but rather add `confirmAction` to https://github.com/RedHatInsights/insights-frontend-components/blob/72aa2d227bc92230f82397a7b90ecd2af51f84d6/src/PresentationalComponents/Wizard/Wizard.js#L45

Reply:
Will do. Is this just for readability, or is there a performance difference as well?

Reply:
Mostly for readability. There might be a slight performance difference, but that won't be noticable either way.